inttests/volume_test.go

```diff
@@ -372,6 +372,150 @@ func TestMapQueryUnmapVolume(t *testing.T) {
 	deleteAllVolumes(t)
 }
 
+func TestMapQueryUnmapSnapshot(t *testing.T) {
```

Comment:
The methods TestMapQueryUnmapSnapshot and TestMapQueryUnmapSnapshotInvalid are perfroming the same functionality just that one is mocking on valid value and other on invalid. Please optimize these calls into one method, as there is redundant code. 
Refer to this test method for mocking on different scenarios: https://github.com/dell/csi-powerflex/blob/main/service/service_unit_test.go#L382

Reply:
Incorporated the review comment in this commit [3740c4e]
Thanks, Rajshree for your valuable feedback.